api/tests/acceptance/application/organizations/organization-controller_test.js

```diff
@@ -204,6 +209,8 @@ describe('Acceptance | Application | organization-controller', () => {
       expect(response.result.data.attributes['can-collect-profiles']).to.equal(true);
       expect(response.result.data.attributes['email']).to.equal('sco.generic.newaccount@example.net');
       expect(response.result.data.attributes['credit']).to.equal(50);
+      // should update image, but does not (bug to fix)
```

Comment:
Commentaire à supprimer ?

Reply:
Non, ça me permet de dire clairement que je commence par ajouter un test qui met en avant le bug
(sinon le passage de `not.equal` à `equal` est bizarre)